api/coreservice.go

```diff
@@ -1041,81 +1041,64 @@ func (core *coreService) BlockByHash(blkHash string) (*block.Store, error) {
 	if err != nil {
 		return nil, errors.Wrap(ErrNotFound, err.Error())
 	}
-	return &block.Store{
+	return &apitypes.BlockWithReceipts{
 		Block:    blk,
 		Receipts: receipts,
 	}, nil
 }
 
-// BlockMetas returns blockmetas response within the height range
-func (core *coreService) BlockMetas(start uint64, count uint64) ([]*iotextypes.BlockMeta, error) {
+// BlockByHeightRange returns blocks within the height range
+func (core *coreService) BlockByHeightRange(start uint64, count uint64) ([]*apitypes.BlockWithReceipts, error) {
```

Comment:
this API change is not necessary? i mean, returning `BlockMeta` is good enough for caller to use?